app/src/app/components/contents/manageAccess/AddRoleForm.tsx

```diff
@@ -23,7 +23,13 @@ export const AddRoleForm = ({ mutate, setOpen }: AddRoleFormProps) => {
   const [fetchError, setFetchError] = useState("");
 
   const formSchema = z.object({
-    name: z.string().min(1),
+    name: z
+      .string()
+      .min(1)
+      .regex(
+        /^[a-zA-Z0-9]+(\s[a-zA-Z0-9]+)*$/,
+        "Name must only contain alphanumeric characters & no leading/trailing spaces."
+      ),
```

Comment:
This might be too restrictive? But I think it's fair to encourage sensible role names!